### ISSUE TITLE
Release 2.7.5

### ISSUE THREAD

**Issue by grosser** [petergoldstein/dalli#580]
@petergoldstein 

current 2.7.4 has a nasty bug, that is fixed in master:

```
    it 'can read incremented key' do
      @dalli = ActiveSupport::Cache.lookup_store(:dalli_store)
      @dalli.with_local_cache do
        @dalli.write('xxx', 0, raw: true)
        @dalli.increment('xxx')
        @dalli.read('xxx').must_equal "1"
      end
    end
```

master: returns "1"
v2.7.4: returns ActiveSupport::Cache::Entry 1

can you bump a 2.8 or a 2.7.5 with just https://github.com/petergoldstein/dalli/pull/481

/cc @sj26

**Comment by petergoldstein** [comment]
@grosser  So I probably won't bump a new version with just #481.  That said, the release of 2.7.5 is overdue at this point.  I need to do some verification, but I can plan to release 2.7.5 (including everything that's currently merged/in the changelog) in the next week.

**Comment by petergoldstein** [comment]
2.7.5 has been released, and includes the fix for #481.

**Comment by grosser** [comment]
Thx!
